eo-maven-plugin/src/main/java/org/eolang/maven/ResolveMojo.java

```diff
@@ -194,6 +194,7 @@ private Collection<Dependency> deps() throws IOException {
             tojo.set(AssembleMojo.ATTR_JAR, coords);
         }
         this.checkConflicts(deps);
+        ResolveMojo.addRuntimeDependency(deps);
```

Comment:
@volodya-lombrozo check this out: https://www.yegor256.com/2015/12/08/temporal-coupling-between-method-calls.html

Reply:
In order to obrain the issue I had to add a few private static classes. Please, take a look at `AllDependencies` and `UniqueDependencies` classes.